### ISSUE TITLE
Button text fields too short for non-latin UTF8 text

### ISSUE THREAD

**Issue by doron1** [Blackymas/NSPanel_HA_Blueprint#987]
Gentlefolks, can you please take a look at #664 again? It was closed as completed but I'm not sure it is. Even 4.0dev has this limitation, which means that the 10-character limit is in fact 10 **bytes**, which in many non-Latin scripts is as little as 5 characters.

**Comment by doron1** [comment]
Multiline is a different thing (may also be important, but different). This one is talking about the fact that 10 bytes in many non-Latin scripts end up being 5 characters, occupying 5 character positions on the screen. So it's not a pixel size thing, it's a field size thing. If you pass e.g. a 20-byte Hebrew or Arabic UTF8-encoded string to the Nextion, you will see 10 characters, occupying similar space to 10 Latin characters. (for those languages you will also see them reversed - due to these scripts being written RTL - but that's a totally different thing). 

You can see a whole discussion about this in #664 .

**Comment by edwardtfn** [comment]
I agree multiline is a different thing, but when adding multiline, thar PR extended the string limit to 22 bytes, which might solve the issue you reported about the 5 char limit when using utf-8.

**Comment by edwardtfn** [comment]
I've applied a change to `dev` extending to 22 bytes on buttons. I couldn't test today, but will appreciate if someone can give a feedback.
#988
